src/material/tabs/tab-content.ts

```diff
@@ -21,5 +21,5 @@ export const MAT_TAB_CONTENT = new InjectionToken<MatTabContent>('MatTabContent'
   providers: [{provide: MAT_TAB_CONTENT, useExisting: MatTabContent}],
 })
 export class MatTabContent {
-  constructor(public template: TemplateRef<any>) { }
+  constructor(public _templateRef: TemplateRef<any>) { }
```

Comment:
This is renaming a public member which is a breaking change.

Reply:
In this case, I've documented it. Please take a look.